lib/service/Service.ts

```diff
@@ -349,6 +350,44 @@ class Service {
     return this.pool.listPeers();
   }
 
+  /**
+   * Get all trades, swaps and order status related to an order.
+   */
+  public listOrderHistory = async (args: { orderId: string }): Promise<{
+    trades: TradeInstance[], swapDeals: SwapDealInstance[], status: OrderStatus}> => {
+    const order = await this.orderBook.getOrder(args.orderId);
+
+    if (order === null) {
+      throw errors.PAIRID_NON_EXISTENT();
+    }
+
+    const trades = await this.orderBook.getTradesByOrderId(order.id, true);
+    const swapDeals = await this.swaps.getDeals(true);
+
+    const isInOrderBook = this.orderBook.getOwnOrder(order.id, order.pairId);
+    let quantity = 0;
```

Comment:
Can we call this `tradedQuantity` or `filledQuantity`? It was confusing to me at first.